Makefile

```diff
@@ -23,6 +23,12 @@ review:
 	$(eval export TF_VAR_app_name=$(APP_NAME))
 	echo https://check-childrens-barred-list-$(APP_NAME).test.teacherservices.cloud will be created in aks
 
+test:
```

Comment:
👍 nice - to be used in conjunction with: https://github.com/DFE-Digital/check-childrens-barred-list/pull/28/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R133